src/feature/hs_common/shared_random_client.c

```diff
@@ -242,8 +238,14 @@ sr_state_get_start_time_of_current_protocol_run(void)
   networkstatus_t *ns = networkstatus_get_live_consensus(approx_time());
```

Comment:
I think we should update the comment above this to reflect the new tristate reality.